app/models/requests/VerifiedEmailRequest.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.requests
+
+import play.api.mvc.WrappedRequest
+
+case class VerifiedEmailRequest[A](request: EORIRequest[A], email: String) extends WrappedRequest[A](request) with Authenticated {
```

Comment:
Would it make sense for the email field to be of type `VerifiedEmailAddress` ?

Reply:
I don't see us ever really needing the value of the timestamp (email maybe if we need to replay it back to the user).